script/OWRV2FactoryScript.s.sol

```diff
@@ -18,7 +18,8 @@ contract OWRV2FactoryScript is Script {
   address constant consolidationSysContract = 0x00431F263cE400f4455c2dCf564e53007Ca4bbBb;
   // From https://github.com/ethereum/EIPs/blob/d96625a4dcbbe2572fa006f062bd02b4582eefd5/EIPS/eip-7002.md#configuration
   address constant withdrawalSysContract = 0x0c15F14308530b7CDB8460094BbB9cC28b9AaaAA;
-
+  // By default the script is aiming devnet-5 (7088110746)
```

Comment:
Given that now we are at devnet-6, I guess the chainid will be different (deposit contract will most likely be the same though).